migrations/versions/4b7fe9b25c6c_removed_legacy_name_from_project.py

```diff
@@ -71,4 +71,5 @@ def downgrade():
 
     for data in legacy_data:
         prof = conn.execute(sa.select([profile.c.id]).where(profile.c.name == data['profile_name']).limit(1)).fetchone()
```

Comment:
I wish you had called this `profile`. I had to scroll up and down to figure out why you were referring to a professor.

Reply:
😓

Reply:
`profile` is already the table name. I'll rename it to something verbose.